### PR TITLE
Mastadon share button

### PR DESCRIPTION
**Pull request by E-Kemp**
Closes https://github.com/elifesciences/enhanced-preprints-issues/issues/585

### CODE REVIEW

**Comment by discodavey** [comment]
Should these icons be on the same line?

Screenshot 2023-03-31 at 12 33 35

File: src/components/atoms/socials/socials.scss [hunk 1]
```diff
@@ -4,7 +4,6 @@
 
 .socials-container {
```
Comment (discodavey) [inline src/components/atoms/socials/socials.scss:5]: If we are not using this class now then we need to remove it

**Comment by nlisgo** [comment]
@chris-huggins are you able to approve?

https://pr-533--epp.elifesciences.org/reviewed-preprints/80494

**Comment by chris-huggins** [comment]
@nlisgo Looks good to me, thanks.

**Comment by nlisgo** [comment]
> Should these icons be on the same line? This is at 320 breakpoint
> 
> Screenshot 2023-03-31 at 12 33 35

@chris-huggins please raise this as a ticket if not working as expected with guidance on how it should be working. Thanks

**Comment by chris-huggins** [comment]
@nlisgo Yes Dave is right. I've raised a ticket here:  https://github.com/elifesciences/enhanced-preprints-issues/issues/586

**Pull request merged**